stdlib/public/Concurrency/TaskSleep.swift

```diff
@@ -14,10 +14,12 @@ import Swift
 
 @available(SwiftStdlib 5.5, *)
 extension Task where Success == Never, Failure == Never {
-  /// Suspends the current task for _at least_ the given duration
+  /// Suspends the current task for at least the given duration
   /// in nanoseconds.
+  /// ◊TR: Why do we have both sleep(_:) and sleep(nanoseconds:) -- what's the difference between them?
+  /// ◊TR: It looks like only sleep(nanoseconds:) throws.
```

Comment:
@ktoso or @DougGregor can you comment?

Reply:
I thought we were going to deprecate `sleep(_:)` so that we could later bring it back when we had a legitimate `Duration` type.  If the argument is just a raw number, then denoting the units as `sleep(nanoseconds:)` is much clearer.

Reply:
That sounds right to me, but I don't see an `@availability` annotation on `sleep(_:)` that marks it as deprecated here or on the [release branch](https://github.com/apple/swift/blob/release/5.5-08092021/stdlib/public/Concurrency/TaskSleep.swift#L15).  It sounds like there should be one?